pkg/descheduler/policyconfig.go

```diff
@@ -122,6 +101,17 @@ func setDefaultEvictor(profile api.Profile) api.Profile {
 		profile.Plugins.Evict.Enabled = append(profile.Plugins.Evict.Enabled, defaultevictor.PluginName)
 		profile.PluginConfigs = append(profile.PluginConfigs, newPluginConfig)
 	}
+	var thresholdPriority int32
+	var err error
+	defaultevictorPluginConfig, idx := GetPluginConfig(defaultevictor.PluginName, profile.PluginConfigs)
+	if defaultevictorPluginConfig != nil {
+		thresholdPriority, err = utils.GetPriorityFromStrategyParams(context.TODO(), client, defaultevictorPluginConfig.Args.(*defaultevictor.DefaultEvictorArgs).PriorityThreshold)
```

Comment:
Main thing to be able to use universal decoder for both conversions was moving this here, and not needing a client in the conversion.go file